react-native-pokedex/src/components/card.jsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { View, Text, StyleSheet, Image } from "react-native";
+import { useFonts } from "expo-font";
+
+
+const Card = (props) => {
+  const [loaded] = useFonts({
+    Montserrat: require('./assets/fonts/Montserrat.ttf'),
+  });
+
+  if (!loaded) {
+    return null;
+  }
+
+  const setStyles = (type) => {
+    return styles[type]
+  }
+
+  return (
+    <View style={styles.card, setStyles(props.type)}>
+      <View style={styles.container}>
+        <Text style={styles.text}>props.pokemon</Text>
```

Comment:
This will render the text "props.pokemon" To access the prop, wrap in curly braces {props.pokemon}